scalyr_agent/log_processing.py

```diff
@@ -926,6 +926,10 @@ def prepare_for_inactivity(self, current_time=None):
         if close_file:
             for pending in self.__pending_files:
                 self.__close_file(pending)
+                log.info(
```

Comment:
Can you add information about what the values are - both of the field, and also the results of the calculation used to trigger this branch? i.e. the value of `delta`, `current_time`, `self.__modification_time_raw` and `self.__max_modification_duration`.

This way we can not only tell what is happening, but also why.

Reply:
The data type in `self.__pending_files` is `FileState`.  Newbie Python question, will it print correctly?

Reply:
It should, if using the `%s` placeholder and you use `%` for formatting, then python will invoke `str()` on the object, which will print a string representation.

Reply:
Worst case scenario is that it will be something like `<FileState object at 0x105e764a8>`